README.md

```diff
@@ -217,23 +217,6 @@ install {
 
 More flags can be found in the `install` section of [the official adb document](https://developer.android.com/studio/command-line/adb.html#pm).
 
-**deviceId**
-
-*Note:* Due to a [bug](https://github.com/novoda/gradle-android-command-plugin/issues/138), setting `deviceId` has been disabled for version `2.0.1`. 
```

Comment:
This was changed for 2.0.1 but now removed completely. I don't think there is a good way to support this feature. `deviceId` can already be overridden overall.